recipe/meta.yaml

```diff
@@ -15,83 +17,178 @@ source:
     - cairo
     - eigen
     - freetype
-    - python
+    - libboost-devel
+    - libboost-python-devel
     - numpy
-    - pillow
     - pandas <2.2
+    - pillow
     - pip
+    - postgresql
+    - python
     - setuptools
     - setuptools_scm >=8
-    # For rdkit-stubs
-    - pybind11-stubgen
-  run:
-    - cairo
-    - python
-    - pillow
-    - pandas
-    - {{ pin_compatible('numpy') }}
-    - pycairo
-    - matplotlib-base
-    - sqlalchemy
-    - reportlab
-
-test:
-  commands:
-    - python -c "import rdkit; assert rdkit.__version__ == '{{ version }}'"
-  imports:
-    - rdkit
-    - rdkit.Avalon
-    - rdkit.Chem
-    - rdkit.Chem.AllChem
-    - rdkit.Chem.rdFreeSASA
-    - rdkit.DataManip
-    - rdkit.Dbase
-    - rdkit.DistanceGeometry
-    - rdkit.ForceField
-    - rdkit.Geometry
-    - rdkit.ML
-    - rdkit.Numerics
-    - rdkit.SimDivFilters
-    - rdkit.VLib
-    - rdkit.VLib.NodeLib
-    - rdkit.Chem.PandasTools
 
 outputs:
+  - name: librdkit
+    script: install.sh   # [unix]
+    script: install.bat  # [win]
+    build:
+      run_exports:
+        - {{ pin_subpackage('librdkit', exact=True) }}
+    requirements:
+      build:
+        - cmake
+      host:
+        - libboost-devel
+      run:
+        - cairo
+        - eigen
+        - freetype
+    test:
+      commands:
+        - test -f $PREFIX/include/rdkit/GraphMol/GraphMol.h    # [unix]
+        - test -f $PREFIX/lib/libRDKitGraphMol.so              # [linux]
+        - test -f $PREFIX/lib/libRDKitGraphMol.dylib           # [osx]
+        - if not exist %LIBRARY_BIN%\RDKitGraphMol.dll exit 1  # [win]
+    about:
+      summary: RDKit C++ library
+      license: BSD-3-Clause
+      license_file: license.txt
+
+  - name: librdkit-dev
+    script: install.bat  # [win]
+    build:
+      skip: True  # [not win]
+    requirements:
+      run:
+        - {{ pin_subpackage('librdkit', exact=True) }}
+    test:
+      commands:
+        - if not exist %LIBRARY_INC%\\rdkit\\GraphMol\\GraphMol.h exit 1  # [win]
+    about:
+      summary: RDKit headers and library used in librdkit (Windows only)
+      license: BSD-3-Clause
+      license_file: license.txt
+
   - name: rdkit
-  - name: rdkit-dev
-    script: install_rdkit_dev.bat  # [win]
+    script: install.sh   # [unix]
+    script: install.bat  # [win]
     requirements:
       build:
-        - {{ stdlib("c") }}
+        - {{ compiler('cxx') }}
+        - {{ stdlib('c') }}
+        - cross-python_{{ target_platform }}  # [build_platform != target_platform]
+        - numpy                               # [build_platform != target_platform]
+        - python                              # [build_platform != target_platform]
+        - cmake
+        - jom   # [win]
+        - make  # [unix]
+        - pkg-config
+      host:
+        - cairo
+        - eigen
+        - freetype
+        - libboost-devel
+        - libboost-python-devel
+        - matplotlib-base
+        - numpy
+        - pandas <2.2
```

Comment:
Why is this pinned but then the `run` counterpart is not? 🤔

Reply:
I'm not sure why the pin is there at all; @greglandrum any reason we should be pinning the pandas version? I'll remove the pin for now and add it to host/run if we decide to keep it.

Reply:
(Note that since this isn't in the run requirements on `main` the current conda package is not being pinned)